js/widgets/temperament.js

```diff
@@ -782,14 +782,28 @@ function TemperamentWidget() {
         this.tempRatios = [];
 
         divAppend.addEventListener("click", function (event) {
-            that.performEqualEdit(event);
+            try{
+                that.performEqualEdit(event);
+            } catch{
+                alert("Number of divisions cannot exceed 999")
+            }
+
         });
 
         this.performEqualEdit = function (event) {
             pitchNumber1 = Number(docById("octaveIn").value);
             pitchNumber2 = Number(docById("octaveOut").value);
             divisions = Number(docById("divisions").value);
             this.tempRatios = this.ratios.slice();
+
+            if(pitchNumber1 > 999 || pitchNumber2 > 999){
```

Comment:
Maybe we just let the error above handle these limits?

Reply:
sure.